### PR TITLE
Test for gather + typo-fixing

### PR DESCRIPTION
**Pull request by huitseeker**

### CODE REVIEW

**Comment by bddppq** [comment]
@onnxbot test this please

File: onnx/backend/test/case/node/gather.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        expect(node, inputs=[data, indices], outputs=[y],
+               name='test_gather_0')
+
+    @staticmethod
+    def export_gather_1():
+        node = onnx.helper.make_node(
+            'Gather',
+            inputs=['data', 'indices'],
+            outputs=['y'],
+            axis=0,
```
Comment (bddppq) [inline onnx/backend/test/case/node/gather.py]: Should here be axis=1?

**Comment by bddppq** [comment]
@huitseeker Everything looks good now except you need to also checkin the new test case's code (i.e. the file node/gather.py).

**Comment by huitseeker** [comment]
@bddppq Yup, sorry, got a bit messed up in the included set for the PR while rewriting the commit. Thanks!

**Comment by bddppq** [comment]
Nice Thank you!

**Pull request merged**